geometric_data/seaice/region/April_Historical_Median_Sea_Ice_Extent/region.geojson

```diff
@@ -0,0 +1,2160 @@
+{
+    "type": "FeatureCollection",
+    "groupName": "enterGroupName",
+    "features": [
+        {
+            "type": "Feature",
+            "properties": {
+                "name": "December Median Sea Ice Extent (1981-2010)",
```

Comment:
It seems like the name got changed to December but the directory is still April.  That shouldn't be possible if you do `gf.split(fc)` to split out the features into the directories in `geometric_data`.  But I think April is probably correct and the change to December was a mistake?

Reply:
Thanks @xylar. That was just a typo copied from another file, I just pushed a fix for it